Lib/defconQt/glyphView.py

```diff
@@ -1359,15 +1664,12 @@ def __init__(self, glyph, parent=None):
         self._showMetricsTitles = True
 
         self.setBackgroundBrush(QBrush(Qt.lightGray))
-        self.setScene(GlyphScene(self))
+        self.setScene(GlyphScene(self, self._getSceneItems('scene-added-items')))
```

Comment:
Okay, we're taking objects and carrying them to a new Scene... isn't a setGlyph approach simpler?